README.md

```diff
@@ -193,6 +193,11 @@ use the `--prefix` argument like so:
 ./bin/install-libs --prefix=/opt/logjam
 ```
 
+If you are using Homebrew as your package manager on Mac OS it is recommended to
+install the libraries with `--without-documentation` to prevent issues arising
+from trying to validate XML files that can seemingly only be validated when
+`docbook-xsl-nons` is installed.
```

Comment:
Even when using this flag it ran into the following here:
```
Making all in doc
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -omakecert.xml makecert.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozactor.xml zactor.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozauth.xml zauth.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozarmour.xml zarmour.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozbeacon.xml zbeacon.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozcert.xml zcert.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozcertstore.xml zcertstore.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozchunk.xml zchunk.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozclock.xml zclock.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozconfig.xml zconfig.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozdigest.xml zdigest.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozdir.xml zdir.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozdir_patch.xml zdir_patch.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozfile.xml zfile.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozframe.xml zframe.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozgossip.xml zgossip.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozhashx.xml zhashx.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -oziflist.xml ziflist.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozlistx.xml zlistx.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozloop.xml zloop.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozmonitor.xml zmonitor.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozmsg.xml zmsg.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozpoller.xml zpoller.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozproxy.xml zproxy.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozrex.xml zrex.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozsock.xml zsock.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozsock_option.xml zsock_option.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozstr.xml zstr.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozsys.xml zsys.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozuuid.xml zuuid.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozauth_v2.xml zauth_v2.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozbeacon_v2.xml zbeacon_v2.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozctx.xml zctx.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozhash.xml zhash.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozlist.xml zlist.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozmonitor_v2.xml zmonitor_v2.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozmutex.xml zmutex.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozproxy_v2.xml zproxy_v2.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozsocket.xml zsocket.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozsockopt.xml zsockopt.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -ozthread.xml zthread.txt
asciidoc -d manpage -b docbook -f ./asciidoc.conf \
        -aczmq_version=3.0.2 -oczmq.xml czmq.txt
xmlto man makecert.xml
xmlto man zactor.xml
xmlto man zauth.xml
xmlto: /Users/sebastian.roebke/work/github.com/skaes/logjam-tools/builds/repos/czmq/doc/zauth.xml does not validate (status 3)
xmlto: /Users/sebastian.roebke/work/github.com/skaes/logjam-tools/builds/repos/czmq/doc/makecert.xml does not validate (status 3)
xmlto: /Users/sebastian.roebke/work/github.com/skaes/logjam-tools/builds/repos/czmq/doc/zactor.xml does not validate (status 3)
xmlto: Fix document syntax or use --skip-validation option
xmlto: Fix document syntax or use --skip-validation option
xmlto: Fix document syntax or use --skip-validation option
I/O error : Attempt to load network entity http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd
/Users/sebastian.roebke/work/github.com/skaes/logjam-tools/builds/repos/czmq/doc/makecert.xml:2: warning: failed to load external entity "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd"
D DocBook XML V4.5//EN" "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd"
                                                                               ^
I/O error : Attempt to load network entity http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd
warning: failed to load external entity "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd"
validity error : Could not load the external subset "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd"
Document /Users/sebastian.roebke/work/github.com/skaes/logjam-tools/builds/repos/czmq/doc/makecert.xml does not validate
I/O error : Attempt to load network entity http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd
/Users/sebastian.roebke/work/github.com/skaes/logjam-tools/builds/repos/czmq/doc/zauth.xml:2: warning: failed to load external entity "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd"
D DocBook XML V4.5//EN" "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd"
                                                                               ^
I/O error : Attempt to load network entity http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd
warning: failed to load external entity "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd"
validity error : Could not load the external subset "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd"
Document /Users/sebastian.roebke/work/github.com/skaes/logjam-tools/builds/repos/czmq/doc/zauth.xml does not validate
I/O error : Attempt to load network entity http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd
/Users/sebastian.roebke/work/github.com/skaes/logjam-tools/builds/repos/czmq/doc/zactor.xml:2: warning: failed to load external entity "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd"
D DocBook XML V4.5//EN" "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd"
                                                                               ^
I/O error : Attempt to load network entity http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd
warning: failed to load external entity "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd"
validity error : Could not load the external subset "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd"
Document /Users/sebastian.roebke/work/github.com/skaes/logjam-tools/builds/repos/czmq/doc/zactor.xml does not validate
make[1]: *** [zauth.3] Error 13
make[1]: *** Waiting for unfinished jobs....
make[1]: *** [zactor.3] Error 13
make[1]: *** [makecert.1] Error 13
rm zsys.xml zconfig.xml zmonitor_v2.xml zauth_v2.xml zhash.xml zfile.xml zarmour.xml zbeacon_v2.xml zthread.xml zhashx.xml zuuid.xml makecert.xml zlistx.xml zgossip.xml zmonitor.xml zframe.xml zsock_option.xml zproxy_v2.xml zsockopt.xml zsocket.xml zdir_patch.xml zclock.xml zdigest.xml zpoller.xml zchunk.xml zactor.xml zbeacon.xml zsock.xml zauth.xml zloop.xml zmutex.xml ziflist.xml zstr.xml zlist.xml czmq.xml zctx.xml zcert.xml zcertstore.xml zrex.xml zproxy.xml zmsg.xml zdir.xml
make: *** [all-recursive] Error 1
```

Reply:
Do you mind trying again with `--without-documentation --forced`?

Reply:
It worked now! Thanks.